### PR TITLE
Add model for resnext101_32x8d_wsl

### PR DESCRIPTION
**Pull request by pellegreene**

### CODE REVIEW

File: brainscore_vision/models/resnext101_32x8d_wsl/__init__.py [hunk 1]
```diff
@@ -3,5 +3,5 @@
 from .model import get_model, get_layers
 
 model_registry['resnext101_32x8d_wsl'] = lambda: ModelCommitment(identifier='resnext101_32x8d_wsl',
-                                                               activations_model=get_model('resnext101_32x8d_wsl'),
+                                                               activations_model=get_model(),
                                                                layers=get_layers('resnext101_32x8d_wsl'))
```
Comment (mschrimpf) [inline brainscore_vision/models/resnext101_32x8d_wsl/__init__.py:7]:
```suggestion
                                                               layers=get_layers())
```

File: brainscore_vision/models/resnext101_32x8d_wsl/model.py [hunk 2]
```diff
@@ -22,26 +20,8 @@ def get_model(name):
 
 
 def get_layers(name):
```
Comment (mschrimpf) [inline brainscore_vision/models/resnext101_32x8d_wsl/model.py:22]:
```suggestion
def get_layers():
```

File: brainscore_vision/models/resnext101_32x8d_wsl/model.py [hunk 3]
```diff
@@ -22,26 +20,8 @@ def get_model(name):
...
-    """
-    A method returning the bibtex reference of the requested model as a string.
-    """
-    return """@inproceedings{mahajan2018exploring,
-              title={Exploring the limits of weakly supervised pretraining},
-              author={Mahajan, Dhruv and Girshick, Ross and Ramanathan, Vignesh and He, Kaiming and Paluri, Manohar and Li, Yixuan and Bharambe, Ashwin and Van Der Maaten, Laurens},
-              booktitle={Proceedings of the European conference on computer vision (ECCV)},
-              pages={181--196},
-              year={2018}
-            }"""
```
Comment (mschrimpf) [inline brainscore_vision/models/resnext101_32x8d_wsl/model.py:43]: why remove the bibtex?

**Review by mike-ferguson — CHANGES_REQUESTED** [review]
This looks like it might be a bit old (one of the first ones converted). Probably needs another quick pass to make it updated

File: brainscore_vision/models/resnext101_32x8d_wsl/model.py (hunk 3 above)
Comment (pellegreene) [reply brainscore_vision/models/resnext101_32x8d_wsl/model.py:43]: Not sure how this got removed here, fixing in next commit.

File: brainscore_vision/models/resnext101_32x8d_wsl/__init__.py (hunk 1 above)
Comment (pellegreene) [reply brainscore_vision/models/resnext101_32x8d_wsl/__init__.py]: Implemented this in last commit.

File: brainscore_vision/models/resnext101_32x8d_wsl/model.py (hunk 2 above)
Comment (pellegreene) [reply brainscore_vision/models/resnext101_32x8d_wsl/model.py]: Implemented this in last commit.

**Review by mike-ferguson — APPROVED** [review]

**Pull request merged**